apps/condo/domains/acquiring/access/AcquiringIntegration.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Generated by `createschema acquiring.AcquiringIntegration 'name:Text;'`
+ */
+
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+
+async function canReadAcquiringIntegrations ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
```

Comment:
wow! we need it everywhere! %)

Reply:
what about isAdmin return {} ?

Reply:
Everybody should be able to access this schema (may be will do field accesses later)